resume-buddy/src/main/java/com/google/sps/servlets/LoginServlet.java

```diff
@@ -33,14 +34,17 @@
 @WebServlet("/login")
 public class LoginServlet extends HttpServlet {
 
+  public static String userType;
```

Comment:
Instead of storing this as a String, create an enum for this type:
```
public static enum UserType{
  REVIEWEE,
  REVIEWER
}
```

Reply:
I'm not sure it would make sense to make it an enum here, only because I retrieve the value of userType this way:
  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
    userType = ServletHelpers.getParameter(request, "user-type", "");
    response.sendRedirect("/index.html");
  }

Reply:
The benefit of using an enum over a string is so you never accidentally input an invalid string for a user type. Eg. you can't accidentally call `addComments("revieweee", "oliviadang@google.com");`. Instead you have to pass in a `UserType` enum to `addComments`.

When getting the user type from the data store, you could just do
```
String userTypeString = ServletHelpers.getParameter(request, "user-type", "");
this.userType = userTypeString == "reviewer" ? UserType.REVIEWER : UserType.REVIEWEE;
```